### PR TITLE
Minification is Working

### LINKED ISSUE #165: fix minification on release
currently enabling proguard on release fails to build. we need to fix this as releases must be minified

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
Fixes #165

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
@championswimmer review

File: app/build.gradle [hunk 1]
```diff
@@ -38,11 +38,11 @@ android {
     buildTypes {
         debug {
             testCoverageEnabled true
-            minifyEnabled false
+            minifyEnabled true
```
Comment (championswimmer) [inline app/build.gradle]: don't enable on debug, that's just to test if it works on not

File: app/proguard-rules.pro [hunk 2]
```diff
@@ -37,11 +37,6 @@
...
-
--keep class io.realm.annotations.RealmModule
--keep @io.realm.annotations.RealmModule class *
--keep class io.realm.internal.** {*;}
--keep @io.realm.internal.Keep class *
--dontwarn javax.**
--dontwarn io.realm.**
\ No newline at end of file
+-keep class android.content.Context {
+    public java.io.File getExternalCacheDir();
```
Comment (championswimmer) [inline app/proguard-rules.pro:41]: aren't Realm classes not needed to be kept ?

Comment (aggarwalpulkit596) [reply app/proguard-rules.pro:41]: Realm is no use of anymore as Room is used everywhere now

File: app/build.gradle (hunk 1 above)
Comment (aggarwalpulkit596) [reply app/build.gradle]: Done

**Pull request merged**